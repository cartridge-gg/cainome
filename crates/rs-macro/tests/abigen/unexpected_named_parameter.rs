#![no_main]
use cainome_rs_macro::abigen;

abigen!(
    MyStruct,
    r#"[
        {
            "type": "struct",
            "name": "core::integer::u256",
            "members": [
              {
                "name": "low",
                "type": "core::integer::u128"
              },
              {
                "name": "high",
                "type": "core::integer::u128"
              }
            ]
        }
    ]"#,
    my_other_parameter(path = "hello")
);
