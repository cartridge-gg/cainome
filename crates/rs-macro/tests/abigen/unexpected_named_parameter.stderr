error: unexpected named parameter `my_other_parameter`
  --> tests/abigen/unexpected_named_parameter.rs:22:5
   |
22 |     my_other_parameter(path = "hello")
   |     ^^^^^^^^^^^^^^^^^^
